ops/model.py

```diff
@@ -26,17 +26,15 @@
 import time
 import typing
 import weakref
-
 from abc import ABC, abstractmethod
 from collections.abc import Mapping, MutableMapping
 from pathlib import Path
-from subprocess import run, PIPE, CalledProcessError
+from subprocess import PIPE, CalledProcessError, run
 
-from ops._private import yaml
-from ops.jujuversion import JujuVersion
 import ops
 import ops.pebble as pebble
-
+from ops._private import yaml
+from ops.jujuversion import JujuVersion
```

Comment:
This doesn't need review -- should have been caught by formatter before this commit.